docs/content/mobile/ios/local-deploy.md

```diff
@@ -63,5 +74,16 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     >
     > For subsequent runs you will be prompted to pick XCode project directory again (with the one used previously preselected). You can generate a new project or append/replace existing one. In subsequent runs, if you've set up provisioning and use Append, you can use **Build and Run** to trigger project run automatically after XCode project was generated.
 
-  1. Once the game is running on your device, you see an empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
+  1. Once the game is running on your device, you need to connect using the local IP address.<br/>Either: 
+  
+      * On your device, note the empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
+      
+      Or:
+  
+      * In your Unity Editor, navigate to: **SpatialOS** > **Launch Mobile Client** > **iOS Client** to launch the iOS client with the IP address field prefilled. (It's prefilled with the value you entered in the GDK Tools Configuration Window. You can access this by navigating to **SpatialOS** > **GDK Tools Configuration** > **Runtime IP for local deployment** in the Unity Editor.)
+  
+      > To use this workflow you need to have [ideviceinstaller](https://github.com/libimobiledevice/ideviceinstaller) and [idevicedebug](https://helpmanual.io/help/idevicedebug/) tools installed.
+      >
+      > To start the iOS client from the Unity Editor, you need to build the application as a `.ipa` archive to the `workers\unity\build` directory.
+
   1. Play the game on your mobile device.
```

Comment:
Suggest slight rejig of the titles.

```suggestion
### Connect your iOS simulator
```


```suggestion
### Connect your iOS mobile device
```

Also, should we have the two mobile device options next to each other, with the simulator last or at the top?